pandas/core/groupby.py

```diff
@@ -2978,7 +2978,8 @@ def is_in_obj(gpr):
 
 def _is_label_like(val):
     return (isinstance(val, compat.string_types) or
-            (val is not None and is_scalar(val)))
+            (val is not None and is_scalar(val)) or
+            isinstance(val, tuple))
```

Comment:
Could the second `isinstance` just be combined with the first, i.e. `isinstance(val, (compat.string_types, tuple))`?

Reply:
Cool, I had no idea of the support for nested tuples (which is undocumented, but apparently [well recognized](https://bugs.python.org/issue25432#msg253502). So OK.